src/stories/Checkbox.js

```diff
@@ -0,0 +1,14 @@
+import { html } from "lit-html";
+
+/**
+ * Primary UI component for user interaction
+ */
+export const Checkbox = ({ disabled, label, checked, isError }) => {
+  return html`
+    <label role="checkbox" aria-invalid="${isError}">
+        <input type="checkbox" .disabled="${disabled}" .checked="${checked}"></input>
+        <span>${label}</span>
+    </label>
+    <span>¡Ouch! Este checkbox no esta tildado.</span>
```

Comment:
Creo que estaría bueno identificar mejor que este componente es un mensaje de error, por ejemplo con una clase:

```suggestion
    <span class="errorMessage">¡Ouch! Este checkbox no esta tildado.</span>
```

Reply:
listo!

Reply:
Se lo había pasado por interno a @GusBaamonde pero para que quede registrado aquí https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-errormessage usar este aria nos podría servir para identificarlo como error.